packages/async-subscription/src/tests/create.test.ts

```diff
@@ -72,6 +72,26 @@ describe('createRemoteSubscribable()', () => {
 
     expect(value).toBe(newValue);
   });
+
+  it('allows to catch async errors in the subscriber callback', async () => {
```

Comment:
With out the changes in this PR this crashes jest

```
 RUNS   async-subscription  packages/async-subscription/src/tests/create.test.ts
node:internal/process/promises:246
          triggerUncaughtException(err, true /* fromPromise */);
          ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "Error: test".] {
  code: 'ERR_UNHANDLED_REJECTION'
}
error Command failed with exit code 1.
```

With the changes it's green.